### DISCUSSION TITLE
3D Print Case

### DISCUSSION THREAD

**Discussion by bjoerns1983** [dokuhn/openDTU-BreakoutBoard#16] [Show and tell]
Just as info i have designed a 3D Printable Case for the Breakout board. If the next testprint is okay, i will send you a PR with the STL and Fusion 360 Files.

![IMG_7941](https://user-images.githubusercontent.com/424566/207355999-0a96ef47-b254-4cde-a081-2915a8b99ad1.JPG)
![IMG_7942](https://user-images.githubusercontent.com/424566/207356008-c00035d8-017d-478c-b036-7417b06b779c.JPG)

**Comment by Bonze255** [comment]
holes for wall mounting on the underside would still be good .. :)

**Comment by dokuhn** [comment]
Thank you for your feedback. The case looks very cool. I will adjust the file structure so that KiCAD files and CAD etc. are separated from each other in directories. This makes it easier with the PR and the linking in the README file.

**Comment by bjoerns1983** [comment]
#7 
Okay PR is created. I have no added any hole for mounting because i dont want the screws to short anything.
Prints quite fast. If any modification is neede the fusion 360 files are included.

**Comment by dokuhn** [comment]
I wish you a happy new year. As you could see, I had the PR included. In the time over Christmas and New Year I have made a bit further on the SMT version of the board. I think we could describe the package a bit more in the README file. Do you want to support the project at this point ? Also a small documentation with two pictures to the Pinout would be probably helpful so other Issues. Let me know if you are up for it.

**Comment by bjoerns1983** [comment]
Yeah no problem, simply tell me what you need.

**Comment by CodeFinder2** [comment]
A version with an OLED display cutout would also be cool :)

**Comment by bjoerns1983** [comment]
I dont have such a board. If you have one, grab the Fusion360 Files, modify them, testprint them and send a PR.

**Comment by huckes** [comment]
Hello, I droped a remix of Bjoerns design with an opening for 1.3" OLED display to Thingiverse: https://www.thingiverse.com/thing:5866231
![IMG_20230220_101703667](https://user-images.githubusercontent.com/35926857/220080896-a751659b-39a7-4a01-aa34-8a724ffcc798.jpg)

**Comment by CodeFinder2** [comment]
Looks like it is an 0,96" display. Do you have a link to the display you used/showed in the photo?

Thank you very much for sharing! 😊

**Comment by bjoerns1983** [comment]
> Hello, I droped a remix of Bjoerns design with an opening for 1.3" OLED display to Thingiverse: https://www.thingiverse.com/thing:5866231 ![IMG_20230220_101703667](https://user-images.githubusercontent.com/35926857/220080896-a751659b-39a7-4a01-aa34-8a724ffcc798.jpg)

Cool, maybe you could drop a PR so it could be added to the Repo?

**Comment by huckes** [comment]
Hi,

@CodeFinder2**:  it is a 1.3" display, rather fast delviered from China: https://de.aliexpress.com/item/4001145494936.html?spm=a2g0o.order_list.order_list_main.19.7e2a5c5fQHz6fJ&gatewayAdapt=glo2deu

@bjoerns1983 : Currently it's just a Q&D remix. I try to refine it a bit (e.g. covering the lower part of the glass where nothng is displayed). When more or less final, I'll drop a PR. In the meanwhile the lid is available under Thingiverse.

I also want to test it completely set up with the hardware, as I'm not certain if the pins of the display come in conflict with the ESP-module.

**Comment by bjoerns1983** [comment]
Ah okay thats why it is tagged WIP.

**Comment by dokuhn** [comment]
very cool job, i am very happy to see cool extensions to the project. Feel free to post a PR when it's tested so we can add it to the repo. This way a lot of people will get something from it.

**Comment by huckes** [comment]
I updated the case with the OLED-opening. Please lookup here: https://www.thingiverse.com/thing:5866226

Some changes to the wiring and the board will follow as PR.
